### ISSUE TITLE
A physical jolt will reboot the module.

### ISSUE THREAD

**Issue by tfabris** [tfabris/BlueGigaEmpeg#51]
Physically bonking the BlueGigaEmpeg prototype unit will make it reboot. Including going over a bump in the car.

I don't know which element is causing it: Arduino, BlueGiga module, power, pololu, USB, what.

But this doesn't make sense because I've made sure there's nothing loose in the assembly.

I've checked and it's not the reset button accidentally getting triggered.

It wasn't the interim wiring on the module as far as I know.

Need to find the root cause of this  before we commit to a final PCB design.

**Comment by tfabris** [comment]
Consider the possibility that this is caused by the two halves of a cut PCB trace just being too close together (the places where you cut traces when you converted from 5v to 3v).

See if opening up those gaps a little more fixes the issue.

**Comment by tfabris** [comment]
Unable to repro on test bench.

See if issue recurs while in the car.

**Comment by tfabris** [comment]
This might be related to issue #55.

**Comment by tfabris** [comment]
Issue #55 has been solved and I'll bet it'll solve this one too. Closing this issue because I'm betting I'll never see it again. If the issue recurs, reopen this card.

**Comment by tfabris** [comment]
This issue has recurred once or twice since I’ve had things in their final mounting locations with final electronics and with good boards without jumper wires or cut traces. Does not happen for small hits on the module, but does happen when car hits a big bad bump. 

Consider that it might be the empeg rebooting because it pops out of the sled a bit. 

Next time it happens see if you can see if the battery icon appears on empeg display screen. If so it’s the empeg and not the BGE, so you can re-close this issue.

**Comment by tfabris** [comment]
I recently found that my Empeg had one of its sled hooks broken and the hook part was rattling around inside my empeg's casing. My bet is that the physical jolt caused this thing to be touching something bad briefly, and rebooting the empeg. Since I haven't had a repro in quite a while, I'm closing this bug again, and assuming that it was due to an empeg reboot all along.

Reopen this issue if you get another repro where you have visually confirmed (by looking for the battery icon on the empeg screen) that it's not an empeg reboot that caused the issue.
